spec/traits.dd

```diff
@@ -939,6 +940,9 @@ $(H2 $(LNAME2 specialkeywords, Special Keywords))
     $(P $(CODE __FILE__) and $(CODE __LINE__) expand to the source
```

Comment:
This doesn't appear to be the case though (I'm only testing on windows, may be different on Linux).  I can go to another directory and compile a source file with a relative path, and the relative path I pass to the compiler isn't maintained:

```
/somedir/test.d

/somedir/build> dmd ../test.d

// __FILE__ will expand to test.d (not ../test.d)
```